### PR TITLE
add productivity wg

### LINKED ISSUE #343: Update doc for new Contributor Productivity WG
as title explains

### PR DESCRIPTION
**Pull request by jessiezcc**
Fixes #343

## Proposed Changes

* added productivity wg description

### CODE REVIEW

File: community/WORKING-GROUPS.md [hunk 1]
```diff
@@ -133,6 +134,24 @@ Slack Channel              | [#autoscaling](https://knative.slack.com)
...
+
+Project health, test framework, continuous integration & deployment, release, performance/scale/load testing infrastructure 
+
+Artifact                   | Link
+-------------------------- | ----
+Forum                      | [knative-dev@](https://groups.google.com/forum/#!forum/knative-dev)
+Community Meeting VC       | [TODO](TODO)
+Community Meeting Calendar | [TODO](TODO)
+Meeting Notes              | [TODO](TODO)
+Document Folder            | [Folder](https://drive.google.com/corp/drive/folders/1vfNC5gXx4VkIW3E3tvXkPVyZuzIs5BgD)
```
[the remark this answers is not in the document]
Comment (jessiezcc) [reply community/WORKING-GROUPS.md]: Got 404, do I have perm?

[the remark this answers is not in the document]
Comment (evankanderson) [reply community/WORKING-GROUPS.md]: Whoops, that should be [`https://drive.google.com/corp/drive/folders/1oMYB4LQHjySuMChmcWYCyhH7-CSkz2r_`](https://drive.google.com/corp/drive/folders/1oMYB4LQHjySuMChmcWYCyhH7-CSkz2r_) -- GitHub didn't linkify the `_`.

**Comment by jessiezcc** [comment]
updated username for previous commits to solve CLA

**Pull request closed without merging**